### ISSUE TITLE
bug: create statement with record id errors

### ISSUE THREAD

**Issue by mathe42** [surrealdb/surrealdb.wasm#3]
Example
![grafik](https://user-images.githubusercontent.com/24830662/193235162-26b7967a-db93-474d-b9cc-889af1ecdf46.png)

True for both statements:
```sql
CREATE chat CONTENT { customer: "guest:ht28nge5czf39lrs39ox" };
CREATE chat SET customer = "guest:ht28nge5czf39lrs39ox"
```

**Comment by mathe42** [comment]
Same for 
```
CREATE c SET A = 42;
```

but not for
```
CREATE c:ex SET A = 42;
```

**Comment by tobiemh** [comment]
@mathe42 can you out down the exact queries/function calls that you used to get to that error after a fresh browser page load?

**Comment by mathe42** [comment]
I used the node build in my vscode extension. It only runs a `USE NS default DB default` before that.

**Comment by tobiemh** [comment]
Ok cool. Where you calling `db.query()` yeah?

**Comment by mathe42** [comment]
Yes.

**Comment by mathe42** [comment]
Just `USE NS default DB default;\nSQL_FROM_ABOVE\n`

**Comment by mathe42** [comment]
Acutaly no I call `db.sql`.

**Comment by tobiemh** [comment]
Hey @mathe42 I can't replicate this...

![Screenshot 2022-10-01 at 15 10 46](https://user-images.githubusercontent.com/1588290/193413546-caff17c3-f65d-4af3-8b9f-9e5e308512f3.png)

**Comment by mathe42** [comment]
Will have a deeper look next week.

**Comment by scotttrinh** [comment]
I'm also running into this with a simple test setup. I'll see if I can put together a minimal repro repo.

**Comment by rushmorem** [comment]
@mathe42 and @scotttrinh Kindly try the latest main branch and let me know if you still run into this. I cannot reproduce it myself.

**Comment by kearfy** [comment]
This is fixed by #17! Thanks for reporting!
